### ISSUE TITLE
USB Ports Turn Off, but don't Passthrough

### ISSUE THREAD

**Issue by termz64** [ra1nstorm/ra1nstorm-helper#8]
Title says it all, finally got this to turn off the ports and open the mac vm, but do nothing when anything is plugged into the ports when in the mac home screen. I have tried to do the KVM manual pass through a couple of days ago and I still am having the exact same problem now. The included [picture](https://imgur.com/OdyzJY7) is the screen the verbose boot hangs at for the longest. When clover boots up, the phone screen lights up indicating it's charging, but when I select the KVMDisk and the verbose boot starts, the ports turn off and stay off

**Comment by termz64** [comment]
I see you put a bunch of tags on here but don't know what's causing the problem. Is there some sort of debugging or logs I can provide you with? Anything u need just let me know, I've been trying to get a linux vm working for Checkra1n for close to a week now

**Comment by andwat18** [comment]
I have the same problem ports not working and I’m using a keyboard lol nightmare I’m on the xUbuntu screen but nothing else happening

**Comment by Tarang74** [comment]
> Title says it all, finally got this to turn off the ports and open the mac vm, but do nothing when anything is plugged into the ports when in the mac home screen. I have tried to do the KVM manual pass through a couple of days ago and I still am having the exact same problem now. The included [picture](https://imgur.com/OdyzJY7) is the screen the verbose boot hangs at for the longest. When clover boots up, the phone screen lights up indicating it's charging, but when I select the KVMDisk and the verbose boot starts, the ports turn off and stay off

Yeah my QEMU is also stuck on the EXACT same line for like 30 minutes! When I get into macOS I can’t do anything because no USB connections can pass through!

**Comment by Tarang74** [comment]
Is there not something you can do with the terminal? For me it shows 
`(qemu)` so I know I can use qemu commands to change USB settings.

Does anyone know which version of qemu is even running, so that I can look at the documentation to see what command to use??

Also what about the clover boot? Maybe you can change the commands from there?
